.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "go-ethereum"]
 	path = go-ethereum
-	url = https://github.com/EspressoSystems/geth-arbitrum.git
+	url = https://github.com/OffchainLabs/go-ethereum.git
 	branch = integration
```

Comment:
Do they also have a branch called "integration"?

Reply:
https://github.com/EspressoSystems/geth-arbitrum/blob/82ad078a28da6ea5b119f9fcaf3c95e224f7038d/params/config_arbitrum.go#L39

Right now the only thing remained in our fork of `geth` is this flag. Not sure if we can remove it or not

Reply:
It seems that everywhere we use this custom flag we also have the chain config in scope. Can we do the checks via the chain config instead?

Reply:
Oh that type is also defined in geth. Nevermind.

Reply:
So I agree I think we can't get rid of our fork.

Reply:
We can remove this flag - it was mostly related to espresso sequencer IMO. I have pushed the changes without this flag  [bfae26b](https://github.com/EspressoSystems/nitro-espresso-integration/pull/214/commits/bfae26bbffd41de78ed777ab919eb4519c7d6606). If my understanding is right, we can prob remove our fork as well.